src/sentry/static/sentry/app/views/settings/organization/general/organizationSettingsForm.jsx

```diff
@@ -22,11 +23,24 @@ const NewOrganizationSettingsForm = React.createClass({
     onSave: PropTypes.func.isRequired,
   },
 
-  mixins: [ApiMixin],
+  mixins: [ApiMixin, OrganizationState],
 
   render() {
     let {initialData, orgId, onSave} = this.props;
 
+    //Only for adding the Flag to 2FA Enforcement. Please remove when the feature is released to the public.
+    if (!this.getFeatures().has('require-2fa')) {
+      let security_panel = organizationSettingsFields.find(function(panel) {
+        return panel.title == 'Security & Privacy';
+      });
+      security_panel.fields.splice(
+        security_panel.fields.findIndex(function(field) {
+          return field.name == 'require2FA';
+        }),
+        1
+      );
+    }
+
```

Comment:
@billyvg refactored the code to be less brittle. Js isn't my primary language, so let me know if I can improve.

Reply:
the idiomatic way to remove an item from an array would be to use `.filter`
so it would be like 
security_panel.fields = security_panel.fields.filter( f => f.name !== 'require2FA')

...... unfortunately, i'm not sure that it's safe to mutate imports like this, because (could be wrong) it will leak across all consumers of the value, so we might need to figure out a different way (deep cloning before mutate, or filtering when we actually map these into components